packages/gatsby/src/redux/actions/public.js

```diff
@@ -1252,6 +1252,19 @@ actions.createJobV2 = (job: JobV2, plugin: Plugin) => (dispatch, getState) => {
   return createJobV2FromInternalJob(internalJob)(dispatch, getState)
 }
 
+actions.addGatsbyImageSourceUrl = (sourceUrl: string) => {
```

Comment:
I don't think this needs to be a public action. We only need this internally right?

Reply:
This one unfortunately needs to be public since only public actions get passed to `gatsbyImageResolver`

Reply:
Makes sense 👍